docs/docs/en/architecture/configuration.md

```diff
@@ -246,7 +246,8 @@ worker.heartbeat.interval|10|worker-service heartbeat interval, the unit is seco
 worker.max.cpuload.avg|-1|worker max CPU load avg, only higher than the system CPU load average, worker server can be dispatched tasks. default value -1: the number of CPU cores * 2
 worker.reserved.memory|0.3|worker reserved memory, only lower than system available memory, worker server can be dispatched tasks. default value 0.3, the unit is G
 worker.groups|default|worker groups separated by comma, e.g., 'worker.groups=default,test' <br> worker will join corresponding group according to this config when startup
-
+worker.tenant.auto.create|true|The tenant is worker's user, which is used by the worker to submit the job. If system does not have this user, it will be automatically created after the parameter is true.
```

Comment:
Is it better to use ```it will be automatically created after the parameter worker.tenant.distributed.user is true.``` instead of ```it will be automatically created after the parameter is true```?